Update PostHog vs Amplitude

andyvan-ph opened the pull request:
## Changes

- Updated intro and "What's the difference?" section 
- Adds price comparison section

andyvan-ph commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -154,55 +153,117 @@ Surveys are great for tracking customer satisfaction scores like Net Promoter Sc
-| **Feature flags**     | 1 million API requests
-| **A/B testing**       | 1 million API requests  
-| **Surveys**           | 250 responses            
+### Analytics pricing
```
@raquelmsmith would like your input on this price comparison section just to make sure it's all correct. I'm adding more of these price comparison segments to emphasize our price advantage more.

Lior539 replied:
This section is very difficult to read tbh. Feels like an entire blog in itself. Can we simplify it?

The whole explaining anonymous vs identified events is too much. Its a difficult concept to wrap my head around

raquelmsmith commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -154,55 +153,117 @@ Surveys are great for tracking customer satisfaction scores like Net Promoter Sc
+
+**PostHog** charges for usage – i.e. analytics events, recorded replays, feature flag API requests, and so on. No credit card is required to signup and use PostHog, but entering a credit card unlocks almost all features and **you'll pay nothing** if you stay within our **generous free tier**
 
 **Amplitude** charges based on monthly tracked users (MTUs). The cost of add owns, such as A/B testing (not available on self-serve plans), group analytics, and unlimited cohorts (self-serve plans are limited to five) isn't disclosed.
```
```suggestion
**Amplitude** charges based on monthly tracked users (MTUs). The cost of add-ons, such as A/B testing (not available on self-serve plans), group analytics, and unlimited cohorts (self-serve plans are limited to five) isn't disclosed.
```

raquelmsmith commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -154,55 +153,117 @@ Surveys are great for tracking customer satisfaction scores like Net Promoter Sc
+    alt="product analytics pricing"
+/>
+
+PostHog is consistently cheaper for both product and web analytics. In the above comparison, we're assuming 75 identified events per monthly tracked user (MTU), which is the typical usage for most PostHog users.
```
I wonder if it's worth breaking this out for B2C vs B2B users. I _think_ B2C will have a much lower event-to-MTU ratio.

andyvan-ph replied:
I like this idea, but trying to avoid much more complexity atm.

raquelmsmith commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -154,55 +153,117 @@ Surveys are great for tracking customer satisfaction scores like Net Promoter Sc
+| 300k                    | $3,150               | $2,091      | <strong class="text-green">$12,708 per year cheaper</strong> |
+
+
+At each tier, PostHog is around 22% to 33% cheaper than Amplitude. Sending more than 75 identified events per MTU will increase the cost of PostHog relative to Amplitude, but we provide numerous ways to control how many events you send and recuce your costs, including the option to send cheaper **anonymous events**.
```
```suggestion
At each tier, PostHog is around 22% to 33% cheaper than Amplitude. Sending more than 75 identified events per MTU will increase the cost of PostHog relative to Amplitude, and sending fewer events will reduce the cost. 

The benefit of charging per-event is that you can control how many events you send per user by only sending important events, whereas there is less desirable control for tracking fewer _users_. We provide [numerous ways](/tutorials/fewer-unwanted-events) to control how many events you send and reduce your costs, including the option to send cheaper **anonymous events**.
```

raquelmsmith commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -154,55 +153,117 @@ Surveys are great for tracking customer satisfaction scores like Net Promoter Sc
+| <strong class="text-15px">100-250 million</strong>  | <span class="text-[15px] font-semibold">$0.0000187</span><span class="text-sm opacity-70">/event</span>  | <span class="text-[15px] font-semibold">$0.0000150</span><span class="text-sm opacity-70">/event</span> | <strong class="text-green">20% cheaper</strong> |
+| <strong class="text-15px">250+ million</strong>     | <span class="text-[15px] font-semibold">$0.0000100</span><span class="text-sm opacity-70">/event</span>  | <span class="text-[15px] font-semibold">$0.0000090</span><span class="text-sm opacity-70">/event</span> | <strong class="text-green">10% cheaper</strong> |
+
+Anonymous events are the default event type. You'll only be charged the identified event rate when you call `posthog.identify`, or to capture an event from an identified user. 
```
```suggestion
Anonymous events are the default event type. You'll only be charged the identified event rate when you call identifying functions like `posthog.identify()` or `posthog.group()`, or capture an event from an already-identified user. 
```

raquelmsmith reviewed (approved):
a few language suggestions but seems largely accurate

Lior539 commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -16,57 +16,50 @@ import { ComparisonTable } from 'components/ComparisonTable'
-Want to know how PostHog and Amplitude compare? If you remember nothing else, remember these two things:
+Amplitude and PostHog are superficially similar. 
 
-1. **Amplitude** is a product analytics tool built for non-technical users and data analysts.
+They both offer product analytics and a range of additional tools, like feature flags, A/B testing, and session replay. 
 
-2. **PostHog** is an all-in-one platform built for engineers that replaces Amplitude _and_ numerous other tools.
+But, deep down, Amplitude is an analytics tool with some extra bits grafted on. In contrast, PostHog is built from the ground up as an all-in-one platform for building successful products. Here's how they compare.
```
INtro is too long IMO. Can we be more to the point?

andyvan-ph replied:
I could go back to just not having an intro again, tbh. It doesn't feel essential to have it.

Lior539 commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -16,57 +16,50 @@ import { ComparisonTable } from 'components/ComparisonTable'
+What does this mean?
 
-### 2. We're built for engineers
+- It means engineers make product decisions.
+- It means our pricing is 100% transparent.  
+- It means they do support – all product teams have a [weekly support hero](/handbook/engineering/support-hero).
+- It means [we ship fast](/changelog) and iterate based on user feedback. 
+- It means we work in the open – [our roadmap](/roadmap) is public, as is [our company handbook](/handbook). 
+- It means start using PostHog without ever jumping on a quick call – see [how we do sales](/sales).
```
Removed "they" as it was unclear who we're talking about. Also its confusing to have "they" and "we"
```suggestion
- It means engineers make product decisions.
- It means our pricing is 100% transparent.  
- It means engineers do support – all product teams have a [weekly support hero](/handbook/engineering/support-hero).
- It means [we ship fast](/changelog) and iterate based on user feedback. 
- It means we work in the open – [our roadmap](/roadmap) is public, as is [our company handbook](/handbook). 
- It means you can start using PostHog without ever jumping on a "quick call" – see [how we do sales](/sales).
```

Lior539 commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -16,57 +16,50 @@ import { ComparisonTable } from 'components/ComparisonTable'
 
-We support all popular (and some unpopular) [SDKs](/docs/libraries), have [extensively documented APIs](/docs/api), and a [powerful SQL query builder](/docs/hogql), so you can analyze data how you want. We make it easy to [test in production](/product-engineers/testing-in-production), conduct [phased rollouts](/tutorials/phased-rollout), run [beta programs](/tutorials/public-beta-program), and so ...
+### 2. We're building a genuine all-in-one platform
 
-### 3. We're open source and transparent
+We want you to spend less time engineering your data and more time engineering your product. You can ditch [Mixpanel](/blog/posthog-vs-mixpanel) and Amplitude for analytics, [Hotjar](/posthog-vs-hotjar) for replays and surveys, and [LaunchDarkly](/blog/posthog-vs-launchdarkly) for feature flags and experiments, with one seamlessly integrated platform. Need a data warehouse as well? We've [got you ...
```
ON the surface Amplitude is also all-in-one. The paragraph didn't really tell me how we're different

Lior539 commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -16,57 +16,50 @@ import { ComparisonTable } from 'components/ComparisonTable'
+### 3. We're cheaper and have a generous free tier
 
-## Comparing PostHog and Amplitude
+We're committed to [sustainably cheap pricing](/blog/analytics-pricing) on all our products. Every user get a [generous free allowance](#free-tiers) every month. Inputting a card unlocks almost every feature, but you'll pay nothing if you stay within the free tiers. See the [price comparison section](#price-comparison) for more.
```
Maybe make this point #2 ? 

```suggestion
We're committed to [sustainably cheap pricing](/blog/analytics-pricing) on all our products. Every company gets a [generous free allowance](#free-tiers) every month. Inputting a card unlocks almost every feature, but you'll pay nothing if you stay within the free tiers. See the [price comparison section](#price-comparison) for more.
```

andyvan-ph replied:
Good idea.

Lior539 commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -16,57 +16,50 @@ import { ComparisonTable } from 'components/ComparisonTable'
-
 <ComparisonTable column1="PostHog" column2="Amplitude">
-  <ComparisonRow column1="1 million events, unlimited insights" column2="100k monthly tracked users, 10 saved insights" feature="Free usage" description="How much free usage do you get each month?" />
+  <ComparisonRow column1="1 million events" column2="50k tracked users (Starter plan only)" feature="Free usage" description="How much free usage do you get each month?" />
```
Can we avoid saying "free usage" twice?

Lior539 commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -81,12 +74,16 @@ There are two key differences here:
+There are two key differences here:
 
-Feature flags make it easy to roll out features to specific users or groups, and [safely test in production](/product-engineers/testing-in-production). Our feature flags are also tightly integrated with other features, so you can target session replays, surveys, and more using existing feature flags. See our guide on the [benefits of feature flags](/product-engineers/feature-flag-benefits-use-cas ...
+1. PostHog supports autocapture, which makes it much faster to implement if you don't want to manually instrument every event. Don't want autocapture? Just [turn it off](/docs/product-analytics/autocapture) and use one of our SDKs – we offer the best of both worlds.
+
+2. PostHog has a [SQL-based query builder](/docs/hogql) for power users, which enables you to generate custom data analyses using PostHog event data. In future, our query builder will let you analyze data as you would in a business intelligence tool, like Metabase, inside PostHog.
```
should this be at the top before the table?
```suggestion
1. PostHog supports [autocapture](/docs/product-analytics/autocapture), which makes it much faster to implement if you don't want to manually instrument every event. Don't want autocapture? Just [turn it off](/docs/product-analytics/autocapture#disabling-autocapture) and use one of our SDKs – we offer the best of both worlds.

2. PostHog has a [SQL-based query builder](/docs/hogql) for power users, which enables you to generate custom data analyses using PostHog event data. In future, our query builder will let you analyze data as you would in a business intelligence tool, like Metabase, inside PostHog.
```

Lior539 commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -99,9 +96,11 @@ Feature flags make it easy to roll out features to specific users or groups, and
 </ComparisonTable>
 
-### Experimentation
+- Feature flags make it easy to roll out features to specific users or groups, and [safely test in production](/product-engineers/testing-in-production). 
+
+- PostHog's feature flags are also tightly integrated with other features, so you can target session replays, surveys, and more using existing feature flags. See our guide on the [benefits of feature flags](/product-engineers/feature-flag-benefits-use-cases) for more.
```
Same comment. Feels a bit backwards to have this after the comparison table

Lior539 commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -154,55 +153,117 @@ Surveys are great for tracking customer satisfaction scores like Net Promoter Sc
-PostHog and Amplitude both offer a wide range of integrations, sources and destinations. PostHog offers more integrations with developer tools, such as [Sentry](/blog/posthog-vs-sentry), Zapier and [Zendesk](/tutorials/zendesk-session-replays), while Amplitude has more integrations with marketing and advertising platforms. See our [data pipeline docs](/docs/cdp) for a full list of PostHog integra ...
+## Price comparison
+
+### Free tier
```
Hmm do we want to lean in very heavily here? Im not sure. I think we're starting to come off as a bit cheap if we focus on free tier too much. I worry this will scare big companies which expect to pay for prodcuts

andyvan-ph replied:
Fair. We cover the free usage in each product section, so we probably don't need to double up here.

Lior539 reviewed (changes requested):
Overall I like the new direction but I feel there were lots of parts that were difficult to parse. Happy to jump on a call to explain in more detail if comments arent clear

andyvan-ph commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -16,57 +16,50 @@ import { ComparisonTable } from 'components/ComparisonTable'
-
 <ComparisonTable column1="PostHog" column2="Amplitude">
-  <ComparisonRow column1="1 million events, unlimited insights" column2="100k monthly tracked users, 10 saved insights" feature="Free usage" description="How much free usage do you get each month?" />
+  <ComparisonRow column1="1 million events" column2="50k tracked users (Starter plan only)" feature="Free usage" description="How much free usage do you get each month?" />
```
```suggestion
  <ComparisonRow column1="1 million events" column2="50k tracked users (Starter plan only)" feature="Free usage" description="Free usage allowance each month?" />
```

andyvan-ph commented:
@Lior539 Have made a bunch of changes based on your feedback:
- Removed the into
- Tweaked the 3 differences and focused more on data warehouse
- Tweaked some wording to avoid saying "cheap"
- Various edits to tighten things up and make it less wordy
- Simplified the price comparison section
- Moved the free tier comparison to the FAQ

Lior539 commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -16,57 +16,33 @@ import { ComparisonTable } from 'components/ComparisonTable'
+- It means you can use PostHog without ever jumping on a "quick call" – see [how we do sales](/sales).
 
-We support all popular (and some unpopular) [SDKs](/docs/libraries), have [extensively documented APIs](/docs/api), and a [powerful SQL query builder](/docs/hogql), so you can analyze data how you want. We make it easy to [test in production](/product-engineers/testing-in-production), conduct [phased rollouts](/tutorials/phased-rollout), run [beta programs](/tutorials/public-beta-program), and so ...
+### 2. Sustainably affordable
```
I dont really like the word "sustainably" here. Makes me think of envirionment/climate stuff. Can we use a different word here and in the paragraph? Maybe something along the lines of "scalable"

andyvan-ph replied:
Some options:

- Affordable, usage-based pricing
- Affordable and transparent pricing
- Fair, usage-based pricing
- Transparent, usage-based pricing

andyvan-ph replied:
I think the second one

Lior539 replied:
I like the second one too

Lior539 commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -81,12 +57,14 @@ There are two key differences here:
 </ComparisonTable>
 
-### Feature flags
+> 💡 **Good to know:** PostHog supports [autocapture](/docs/product-analytics/autocapture), which means you can implement PostHog in mere minutes and ensure you don't miss out on events you haven't manually instrumented. Don't want autocapture? Just [turn it off](/docs/product-analytics/autocapture#disabling-autocapture) – we offer the best of both worlds.
```
I love all these callouts

Lior539 commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -115,30 +97,34 @@ PostHog's offers a generous free 1 million A/B testing events. Amplitude's A/B t
 </ComparisonTable>
 
-### Surveys
+> 💡 **Good to know:** Replays let you watch how users experience your app, diagnose issues, improve support, and understand real user behavior in a way raw data can't. They are reconstructions of the session, _not_ video recordings of user's screen. Private info, such as passwords, is masked.
```
I think it should technically be "is" but I think "are" is better after reading it after "passwords"
```suggestion
> 💡 **Good to know:** Replays let you watch how users experience your app, diagnose issues, improve support, and understand real user behavior in a way raw data can't. They are reconstructions of the session, _not_ video recordings of user's screen. Private info, such as passwords, are masked.
```

Lior539 commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -154,9 +140,62 @@ Surveys are great for tracking customer satisfaction scores like Net Promoter Sc
+
+They're useful if you're tracking a huge number of users and only want to analyze aggregate data, such as on a content website with millions of visitors. 
+
+Anonymous events cheaper for us to ingest, so we charge you between 55% and 80% less for them. You'll only be charged the identified event rate when you call identifying functions like `posthog.identify()` or `posthog.group()`, or capture an event from an already-identified user. 
```
```suggestion
Anonymous events are cheaper for us to ingest, so we charge you between 55% and 80% less for them. You'll only be charged the identified event rate when you call identifying functions like `posthog.identify()` or `posthog.group()`, or capture an event from an already-identified user. 
```

Lior539 commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -154,9 +140,62 @@ Surveys are great for tracking customer satisfaction scores like Net Promoter Sc
+
+We also provide numerous ways to [control how many events you send](/tutorials/fewer-unwanted-events) and reduce your costs, including the option to send cheaper **anonymous events**.
+
+### Identified vs anonymous events
```
this section is way better now

Lior539 commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -211,14 +221,36 @@ Eligible companies can also get $50,000 in credit in our [PostHog for Startups p
+
+Companies that qualify for [our startup program](/startups) also get $50,000 in PostHog credit and a range of additional benefits. 
+
+### Can PostHog replace Google Analytics?
```
Should this be Amplitude?

Lior539 reviewed (approved):
Looks great!

andyvan-ph commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -1,6 +1,6 @@
 ---
 title: 'In-depth: PostHog vs Amplitude'
-date: 2024-01-08
+date: 2024-09-05
```
```suggestion
date: 2024-09-10
```

andyvan-ph commented on contents/blog/posthog-vs-amplitude.mdx:
```diff
@@ -16,57 +16,33 @@ import { ComparisonTable } from 'components/ComparisonTable'
+### 3. Analytics and data warehouse in one
 
-## Comparing PostHog and Amplitude
+Use our [built-in data warehouse](/data-warehouse) and SQL insight builder to analyze data from [Stripe](/tutorials/stripe-reports), [Hubspot](/tutorials/hubspot-reports), Salesforce, and [Zendesk](/tutorials/stripe-reports). Already have a warehouse? You can [link it with ours](/docs/data-warehouse/setup) to analyze your data in PostHog, and [batch export event and person data](/docs/cdp/batch-e ...
```
```suggestion
Use our [built-in data warehouse](/data-warehouse) and SQL insight builder to analyze data from [Stripe](/tutorials/stripe-reports), [Hubspot](/tutorials/hubspot-reports), Salesforce, and [Zendesk](/tutorials/stripe-reports). 

Already have a warehouse? You can [link it with ours](/docs/data-warehouse/setup) to analyze your data in PostHog, and [batch export event and person data](/docs/cdp/batch-exports) from PostHog to your data warehouse.
```
